packages/app/src/systems/Account/services/account.ts

```diff
@@ -175,6 +174,18 @@ export class AccountService {
     return exitsAccountWithName;
   }
 
+  static async generateAccountName(randomSuffix?: boolean): Promise<string> {
+    const suffix = randomSuffix ? Math.floor(Math.random() * 100) : '';
+    const accounts = await AccountService.getAccounts();
+    const count = accounts.length;
+    const name = `Account ${count + 1}${suffix}`;
+    const accountNameExists = await this.checkAccountNameExists(name);
+    if (accountNameExists) {
+      return this.generateAccountName(true);
+    }
+    return name;
+  }
```

Comment:
this can use `getUniqueString`

it will add (1), (2), (3) at the end if has other account with same name.
`Account 2 (1)`
instead of current "random suffix" -> `Account 2 44` for example

something along this lines \\/

```suggestion
  static async generateAccountName() {
    const accounts = await AccountService.getAccounts();
    const count = accounts.length;
    const desiredName = `Account ${count + 1}`;
    const allNames = accounts.map(({ name }) => name);
    const name = getUniqueString(desiredName, allNames);
  }
```

Reply:
Wow, thanks. I didn't know we had this function, thanks.